specs/core/0_beacon-chain.md

```diff
@@ -1156,20 +1156,45 @@ def slash_validator(state: BeaconState,
 
 ## Genesis
 
-### `Eth2Genesis`
+### Genesis trigger
+
+Whenever the deposit contract emits a `Deposit` log call the function `is_genesis_trigger(deposits: List[Deposit], timestamp: uint64) -> bool` where:
+
+* `deposits` is the list of all deposits, ordered chronologically, up to and including the deposit triggering the latest `Deposit` log
+* `timestamp` is the Unix timestamp in the Ethereum 1.0 block that emitted the latest `Deposit` log
+
+When `is_genesis_trigger(deposits, timestamp) == True` for the first time let:
+
+* `genesis_deposits = deposits`
+* `genesis_time = timestamp % SECONDS_PER_DAY + 2 * SECONDS_PER_DAY` where `SECONDS_PER_DAY = 86400`
```

Comment:
I did not put `SECONDS_PER_DAY ` in the constants because the whole "Genesis trigger" section should probably go in the `0_fork-choice.md` file. This is an intermediate step. I will migrate everything (including the `SECONDS_PER_DAY` constant, putting it in a table) once #1068 is merged.